APITesters/SwiftAPITester/SwiftAPITester/PurchasesAPI.swift

```diff
@@ -206,6 +206,8 @@ private func checkAsyncMethods(purchases: Purchases) async {
         let _: CustomerInfo = try await purchases.restoreTransactions()
         let _: CustomerInfo = try await purchases.syncPurchases()
 
+        for try await _: CustomerInfo in purchases.customerInfoStream {}
```

Comment:
😻  so cool. maybe we can update MagicWeather and/or purchase tester with this version?

Reply:
I looked into it for MagicWeather. The current API made more sense right now I think because of the way it's wrapped into an `ObservableObject`. I think we could change it in `PurchaseTester` though let me take a look.

Reply:
Yeah it would actually simplify `PurchaseTester` a little bit, but it would require making it iOS 13 only. I think that would be okay, but what do you think? /cc @aboedo

Reply:
oops, accidentally clicked "resolve". I think it's totally fine to have MagicWeather be iOS 13+

Reply:
I meant `PurchaseTester` though, but I think same applieS?

Reply:
sorry, my bad. `PurchaseTester` is exclusively for us to use for testing, so no worries about updating its deployment target as long as it's useful